### PR TITLE
Erasing windows update option in wrong cases

### LINKED ISSUE #3566: Windows updates in Linux agant inventory
| Wazuh | Elastic | Rev |
| ----- | ------- | --- |
| 4.x   | 7.x     | - |

**Description**
A section with `Windows updates`  is displayed in reports of a Linux agent inventory.

**Steps to reproduce**

1. Go to a Linux agent
2. Click on the inventory
3. Generate report
4. See the Windows updates section 

**Screenshots**
image

### PR DESCRIPTION
**Pull request by CPAlejandro**
Hi guys,

We disabled Windows Update part of Inventory PDF in agents with an operative system different from Windows.

To test it, go to Agents, click an agent without Windows, go to Inventory Section and generate a PDF. Then go to the end of that and see if Windows Update section appers.

If you can, try as well with a Windows agent to check that in this case, this section appears.

Closes #3566

### CODE REVIEW

**Review by mpRegalado — APPROVED** [review]
CR: LGTM :heavy_check_mark: 
Testing: LGTM :heavy_check_mark:

File: server/controllers/wazuh-reporting.ts [hunk 1]
```diff
@@ -1906,7 +1906,7 @@ export class WazuhReportingCtrl {
             ]
           }
         },
-        {
+        agentOs === 'windows' && {
```
Comment (frankeros) [inline server/controllers/wazuh-reporting.ts:1909]: this is a bad implementation, is not a good practice using a ternary in an array definition 
please use a different approach, maybe inserting after the declaration of the array with a conditional 

also, check if this endpoint is used for another SO but windows

**Review by frankeros — CHANGES_REQUESTED** [review]

File: server/controllers/wazuh-reporting.ts (hunk 1 above)
Comment (CPAlejandro) [reply server/controllers/wazuh-reporting.ts]: I change it using your suggestions. Thanks you!

**Review by frankeros — APPROVED** [review]
LGTM!

**Pull request merged**